src/app.js

```diff
@@ -60,11 +60,14 @@ app.use(passport.initialize());
 app.use(passport.session());
 
 // Database setup (using Mongoose)
+mongoose.set("strictQuery", true);
 
 const connectDB = (url) => {
   return mongoose.connect(url);
 };
 
+module.exports = connectDB;
```

Comment:
Why do we need to  export this? Will other files/modules be needing this `connectDB` function? I think we can just remove it for now if it's not being used yet.